pkg/framework/host.go

```diff
@@ -61,17 +61,17 @@ Installation:
 `
 )
 
+type HostConfig struct {
+	Backoff *backoff.ExponentialBackOff
+}
+
 type Host struct {
 	backoff   *backoff.ExponentialBackOff
 	g8sClient *giantclientset.Clientset
 	k8sClient kubernetes.Interface
 }
 
-type Config struct {
-	Backoff *backoff.ExponentialBackOff
-}
-
-func NewHost(c *Config) (*Host, error) {
+func NewHost(c HostConfig) (*Host, error) {
```

Comment:
I do not know why this should be a pointer. We are not doing it like this. Nowhere.

Reply:
AFAIK changing it to a value can't brake anything in this case.